lib/install.js

```diff
@@ -156,7 +150,7 @@ function install (gyp, argv, callback) {
   }
 
   function getContentSha(res, callback) {
-    var shasum = crypto.createHash(checksumAlgo)
+    var shasum = crypto.createHash(release.checksumAlgo)
```

Comment:
no, that's not checked here anyway, it's in process-release.js and boils down to

```js
  var checksumAlgo = semver.gte(version, '0.10.0') ? 'sha256' : 'sha1'
```

that's all this is for because there are no SHASUM256.txt files in 0.8.x dirs